conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollections.java

```diff
@@ -71,4 +72,22 @@ public static <T> LinkedHashSet<T> newLinkedHashSet(Iterable<? extends T> iterab
         }
         return set;
     }
+
+    // explicitly need to return mutable list for generated builders
+    public static DoubleArrayList newDoubleArrayList(Iterable<Double> iterable) {
```

Comment:
Decided to put all the new primitive array list methods here to minimize complexity in the codegen classes